src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+quarkus.datasource.db-kind=postgresql
+quarkus.datasource.username=corpoapp
+quarkus.datasource.password=corpoapp
+quarkus.datasource.jdbc.driver=org.postgresql.Driver
+quarkus.datasource.jdbc.url=jdbc:postgresql://postgres:5432/corpoapp
```

Comment:
change postgres to corpo-app_postgres  (we have to put the service name of the compose here)